libs/expo/betterangels/src/lib/screens/Client/ClientHeader.tsx

```diff
@@ -1,94 +1,132 @@
+          {client?.nickname && `(${client.nickname})`}
         </TextMedium>
       </View>
-      <ScrollView
-        contentContainerStyle={{ alignItems: 'center', gap: Spacings.sm }}
-        horizontal
-        style={{ flexDirection: 'row' }}
+      <View
+        style={{ flexDirection: 'row', alignItems: 'center', gap: Spacings.xs }}
       >
-        {gender && (
-          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
-            <GenderIcon mr="xs" color={Colors.PRIMARY_EXTRA_DARK} />
-            <TextRegular>{enumDisplayGender[gender]}</TextRegular>
+        {client?.preferredLanguage && (
+          <View
+            style={{
+              flexDirection: 'row',
+              alignItems: 'center',
+              gap: Spacings.xxs,
+            }}
+          >
+            <GlobeIcon color={Colors.PRIMARY_EXTRA_DARK} />
+            <TextRegular>
+              {enumDisplayLanguage[client.preferredLanguage]}
+            </TextRegular>
           </View>
         )}
-        {preferredLanguage && (
-          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
-            <GlobeIcon mr="xs" color={Colors.PRIMARY_EXTRA_DARK} />
+        {client?.pronouns && (
+          <View
+            style={{
+              flexDirection: 'row',
+              alignItems: 'center',
+              gap: Spacings.xxs,
+            }}
+          >
+            <UserIcon color={Colors.PRIMARY_EXTRA_DARK} />
 
-            <TextRegular>{enumDisplayLanguage[preferredLanguage]}</TextRegular>
+            <TextRegular>{client.displayPronouns}</TextRegular>
           </View>
         )}
-        {dateOfBirth && (
-          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
-            <BirthdayIcon mr="xs" color={Colors.PRIMARY_EXTRA_DARK} />
-            <TextRegular>
-              {dateOfBirth} ({age})
-            </TextRegular>
-          </View>
-        )}
-      </ScrollView>
+      </View>
+      <View
+        style={{
+          flexDirection: 'row',
+          alignItems: 'center',
+          gap: Spacings.xxs,
+        }}
+      >
+        <PersonIcon color={Colors.PRIMARY_EXTRA_DARK} />
+        <TextRegular>
+          Current Case Manager:{' '}
+          <TextRegular
+            onPress={() => {
+              if (client?.displayCaseManager !== 'Not Assigned') {
+                return onCaseManagerPress();
+              }
+              return null;
+            }}
+            textDecorationLine="underline"
+          >
+            {client?.displayCaseManager}
+          </TextRegular>
+        </TextRegular>
+      </View>
+      {/* // TODO: last interaction was split out to DEV-761*/}
+      {/* <View
+        style={{
+          flexDirection: 'row',
+          alignItems: 'center',
+          gap: Spacings.xxs,
+        }}
+      >
+        <TentIcon color={Colors.PRIMARY_EXTRA_DARK} />
+        <TextRegular>Last Interaction: N/A</TextRegular>
+      </View> */}
```

Comment:
let's take this out, since I'm not sure when 761 will be done. The comment on line 107 is a good reminder to have. I just don't want to get into the habit of committing commented out code

Reply:
done